src/main/java/com/massivecraft/factions/zcore/fupgrades/FUpgradesGUI.java

```diff
@@ -63,7 +63,7 @@ public void onClick(InventoryClickEvent e) {
 			return;
 
 		FPlayer fme = FPlayers.getInstance().getByPlayer((Player) e.getWhoClicked());
-		if (e.getClickedInventory().getTitle().equalsIgnoreCase(SavageFactions.plugin.color(SavageFactions.plugin.getConfig().getString("fupgrades.MainMenu.Title").replace("{faction}", fme.getFaction().getTag())))) {
+		if (e.getView().getTitle().equalsIgnoreCase(SavageFactions.plugin.color(SavageFactions.plugin.getConfig().getString("fupgrades.MainMenu.Title").replace("{faction}", fme.getFaction().getTag())))) {
```

Comment:
Will this getView work across 1.8 to 1.14? as this is a new addition, if the getClickInventory is just deprecated, either leave the old call, as it will work on all versions.

Reply:
Inventory.getTitle() was removed in this 1.14 update.
InventoryView.getTitle() existed in 1.13 version so i think it's backward compatible.

Reply:
```suggestion
                String title;
		if(SavageFactions.plugin.mc114) {
			title = e.getView().getTitle();
		} else {
			title = e.getClickedInventory().getTitle();
		}
		if (title.equalsIgnoreCase(...
```
I can do this, because older version will run on an other Spigot version and will accept it but the project will not accept to compile because the getTitle() method was removed in our Spigot 1.14 version

Reply:
After some search i found that InventoryEvent haved the getView() method in Spigot 1.8 and InventoryView haved the getTitle() too.
http://docs.codelanx.com/Bukkit/1.8/org/bukkit/inventory/InventoryView.html

Reply:
Perfect.